backend-hormonia/app/routers/auth_session.py

```
@@ -405,7 +421,7 @@ async def validate_session(
             from sqlalchemy import select
 
             stmt = select(User).where(User.firebase_uid == firebase_uid)
-            result = services.db.execute(stmt)
+            result = db.execute(stmt)
```

Pylint (3.3.9)

[error] Undefined variable 'db'

(E0602)